src/App.tsx

```diff
@@ -1,24 +1,221 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Status } from './types/Status';
+import {
+  addTodo,
+  getTodos,
+  deleteTodo,
+  updateTodo,
+} from './api/todos';
 
-const USER_ID = 0;
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
+
+const USER_ID = 7011;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [title, setTitle] = useState('');
+  const [selectedStatus, setSelectedStatus] = useState<Status>(Status.All);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isDisabledInput, setIsDisabledInput] = useState(false);
```

Comment:
I guess you may remove it from the state and calculate from othe state variables:

example:
`const isDisabledInput = tempTodo || !!loadingTodoIds.length;`

and probably similar with `isDeletingCompleted`